libnavigation-base/src/main/java/com/mapbox/navigation/base/options/OnboardRouterOptions.kt

```diff
@@ -7,13 +7,16 @@ import java.net.URI
  * This allows the navigator to map-match your location onto the road network without a route.
  *
  * @param tilesUri tiles endpoint
- * @param tilesVersion version of tiles
+ * @param tilesVersion version of tiles. If not specified (empty), latest version is used
  * @param filePath used for storing road network tiles
+ * @param keepOlderTilesVersions true if older version tiles must be cleaned, false if not. Note
```

Comment:
the comment and the name of the variable contradict (keep != clean)

Reply:
Correct that's what we're discussing https://github.com/mapbox/mapbox-navigation-android/pull/3931#discussion_r572080890 😅

Reply:
get it